test/e2e/reporter.go

```diff
@@ -123,21 +184,32 @@ func (r *KubernetesNMStateReporter) logPods(testName string, namespace string, s
 			io.WriteString(writer, formattedLogs)
 		}
 	})
+
+	// Let's print the pods logs to the GinkgoWriter so
+	// we see the failure directly at prow junit output without opening files
+	if podsLogFile != "" {
+		podsLog, err := ioutil.ReadFile(podsLogFile)
+		if err != nil {
+			fmt.Println(err)
+			return nil
+		}
+		GinkgoWriter.Write(podsLog)
```

Comment:
I think that if test didn't fail there is no need to print logs to stdout for sure

Reply:
Yet, I think we want to have the logs collected to a file, so we will have the option to compare the success logs with failed ones

Reply:
We run this only on failure, so it will be write only then.